RuntimeUnityEditor/Windows/Profiler/ProfilerWindow.cs

```diff
@@ -167,7 +173,10 @@ protected override void DrawContents()
                                 if (!ran && !pd.Owner) _needResort = true;
 
                                 GUILayout.Toggle(ran, GUIContent.none); // enabled
-                                GUILayout.Toggle(pd.OriginalRan, GUIContent.none, _cRanW2); // enabled
+                                if (!_aggregation)
+                                    GUILayout.Toggle(pd.OriginalRan, GUIContent.none, _cRanW2); // enabled
+                                else
+                                    GUILayout.Space(RanW);
```

Comment:
```suggestion
                                    GUILayout.Space(RanW + 4);
```
UI offset is not taking margins into account